### ISSUE TITLE
Unability to freeze the website if no runs have calendar

### ISSUE THREAD

**Issue by hroncok** [pyvec/naucse.python.cz#460]
No run with times (can happen):

    Error: Nothing frozen for endpoints course_calendar_ics. Did you forget a URL generator?

No run at all (unlikely):

    Error: Nothing frozen for endpoints course_calendar, course_calendar_ics. Did you forget a URL generator?

How do we get rid ourselves form this error? We can filter MissingURLGeneratorWarning, but only some.

**Comment by hroncok** [comment]
czechitas-ostrava-jaro is still in master and has calendar. once removed, this will hit me.

**Comment by hroncok** [comment]
@mikicz can this be solved in NaucseFreezer? Do you have some elegant idea?

**Comment by mikicz** [comment]
Well, I have a solution, IDK if it's ideal. You can override the `_check_endpoints` method of the `Freezer` class to do nothing and the problem goes away, it works even without the `czechitas-ostrava-jaro` run...

I also tried to use `warnings.simplefilter` to filter out the warning and it didn't help...

**Comment by hroncok** [comment]
Can we copy paste it and remove specific endpoints from the check?

Maybe we can send PR to Frozen-Flask to do this in a more flexible fashion?

**Comment by mikicz** [comment]
Yes, quite easily, you just need to add one line to the function...

```
not_generated_endpoints -= {'course_calendar_ics', 'course_calendar'}
```

A PR for flexible solution would of course be better.
